pygmt/src/grdlandmask.py

```diff
@@ -44,17 +50,14 @@ def grdlandmask(outgrid: str | None = None, **kwargs) -> xr.DataArray | None:
     {spacing}
     {region}
     {area_thresh}
-    resolution : str
-        *res*\[\ **+f**\]. Select the resolution of the data set to use
-        ((**f**)ull, (**h**)igh, (**i**)ntermediate, (**l**)ow, or
-        (**c**)rude). The resolution drops off by ~80% between data sets.
-        [Default is **l**]. Append **+f** to automatically select a lower
-        resolution should the one requested not be available
-        [abort if not found]. Alternatively, choose (**a**)uto to automatically
-        select the best resolution given the chosen region. Note that because
-        the coastlines differ in details a node in a mask file using one
-        resolution is not guaranteed to remain inside [or outside] when a
-        different resolution is selected.
+    resolution
```

Comment:
Copied from https://docs.generic-mapping-tools.org/dev/grdlandmask.html#d, but the `+f` modifier is not mentioned.